modules/apps/batch-engine/batch-engine-test/src/testIntegration/java/com/liferay/batch/engine/internal/exportimport/data/handler/test/BatchEnginePortletDataHandlerTest.java

```diff
@@ -700,6 +824,10 @@ private void _testExportImportObjectEntriesToSameGroup(
 		File larFile = _exportLayouts(
 			false, group.getGroupId(), false, objectDefinition);
 
+		_inspectLARFile(
```

Comment:
I would prefer not to inspect the lar file as it should be a blackbox for the user. We should assert on the final state of the system once the lar file has been imported.